orchestration/dev/coins/monero/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
```

Comment:
the alpine image doesn't have a bash by default

Reply:
Fun. Guess I'll consolidate to /sh :sweat_smile:  Thanks.

I pushed these changes as I went to bed, will review the accumulated CI errors now.